sf-pcap-ng.c

```diff
@@ -303,7 +303,7 @@ read_block(FILE *fp, pcap_t *p, struct block_cursor *cursor, char *errbuf)
 	/*
 	 * Is the buffer big enough?
 	 */
-	if (p->bufsize < bhdr.total_length) {
+	if ((bpf_u_int32) p->bufsize < bhdr.total_length) {
```

Comment:
This should no longer be necessary with 2d96be0322d71ce62e2f68d038070b10a7c9201b.